build/accelerate_launch.py

```diff
@@ -115,6 +119,54 @@ def main():
         write_termination_log(f"Unhandled exception during training. {e}")
         sys.exit(INTERNAL_ERROR_EXIT_CODE)
 
+    peft_method = job_config.get("peft_method")
+
+    if job_config.get("lora_post_process_for_vllm") and peft_method == "lora":
+        if job_config.get("save_model_dir"):
+            save_model_dir = job_config.get("save_model_dir")
+            if os.path.exists(os.path.join(save_model_dir, "added_tokens_info.json")):
+                with open(
+                    os.path.join(save_model_dir, "added_tokens_info.json"),
+                    encoding="utf-8",
+                ) as json_data:
+                    added_tokens_info = json.load(json_data)
+                    num_added_tokens = added_tokens_info["num_new_tokens"]
+            else:
+                logging.warning(
+                    "file added_tokens_info.json not in model_path. Cannot post-processes"
```

Comment:
Would be helpful to give the model_path
```suggestion
                    "Failed to post-process: file added_tokens_info.json not in path %s", save_model_dir
```